README.md

```diff
@@ -32,15 +32,25 @@ $ npm install
 ### The Design System:
 
 1. Run `npm start`
-2. Assets are then available at http://localhost:9032/assets/v4-SNAPSHOT/...
-3. The Design System is available at http://localhost:9034/
+2. The Design System is available at http://localhost:9034/
 
 ### The Component Library [DEPRECATED]
 
-1. Run `npm run build:all`
-2. Assets are then available at http://localhost:9032/assets/v3-SNAPSHOT/...
+1. Run `./server.sh dev`
```

Comment:
Don't need this command. This just runs `npm run dev:all`

Reply:
Could maybe include this in the section below to say run `./server.sh dev` or `npm run dev:all`